### DISCUSSION TITLE
Node.js 18

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#795] [General]
半年之后（22-10-25）成为 LTS 版本
https://nodejs.org/en/blog/announcements/v18-release-announce/

主要特性：

- **全局 Fetch API** （基于 undici）：全局增加了 fetch, FormData, Headers, Request, Response 几个 API，以及 Web Streams API
- **[Test Runner](https://nodejs.org/dist/latest-v18.x/docs/api/test.html)**
- **[Build-time user-land snapshot](https://nodejs.org/en/blog/announcements/v18-release-announce/#build-time-user-land-snapshot-experimental)**：可以直接打包 js 文件为 Node.js 二进制文件
- v8 引擎（JS处理引擎）升级到 10.1，支持 `Array.findLast()` 和 `Array.findLastIndex()`，其他的看不懂

**Comment by Arondight** [comment]
老实说只要是向下兼容的，对我们的项目木得影响……

**Comment by Arondight** [comment]
发现一个好玩的东西 https://lesscss.org/#nesting

**Reply by mark9804** [reply]
emm，个人不太喜欢 nesting 这种写法，感觉有点乱，不过 sass/scss/less 本身确实提供了一些纯 css 很难实现的事情，比如说 [Parallax Star background in CSS](https://codepen.io/sarazond/pen/LYGbwj) 要是手搓 css 的话会出人命
